aiida_raspa/utils/base_parser.py

```diff
@@ -41,16 +41,20 @@ def float(number):  # pylint: disable=redefined-builtin
 
 # pylint: disable=too-many-arguments
 def parse_block1(flines, result_dict, prop, value=1, unit=2, dev=4):
-    """Parse block that looks as follows:
-    Average Volume:
-    =================
-        Block[ 0]        12025.61229 [A^3]
-        Block[ 1]        12025.61229 [A^3]
-        Block[ 2]        12025.61229 [A^3]
-        Block[ 3]        12025.61229 [A^3]
-        Block[ 4]        12025.61229 [A^3]
-        ------------------------------------------------------------------------------
-        Average          12025.61229 [A^3] +/-            0.00000 [A^3]
+    """Parse volume block.
+
+    Parse block that looks as follows::
```

Comment:
Here it is not only about the volume block. There are many other blocks that have the same structure.